modules/Makefile.circleci

```diff
@@ -8,9 +8,14 @@ BUILD ?= $(CIRCLE_BRANCH)-$(CIRCLE_BUILD_NUM)
 endif
 
 COMMIT ?= $(CIRCLE_SHA1)
+COMMIT_LOG ?= $(shell git log --format=oneline --pretty=format:'%s' -n 1 $(CIRCLE_SHA1))
```

Comment:
It's so we can reference this in the annotations when doing deployments. 

e.g. 
![image](https://cloud.githubusercontent.com/assets/52489/15260366/53653de4-190b-11e6-9f77-57d662cf1669.png)

Not currently necessary since I removed the annotation block from the resource file, but it will hopefully be available soon given the rate they iterate & release kubernetes. They are actively working on deployments & annotations (per @janetkuo).